### ISSUE TITLE
Crash in android 10

### ISSUE THREAD

**Issue by Farhandroid** [esafirm/android-image-picker#313]
App crashes with bellows log report

```
    java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter folderName
        at com.esafirm.imagepicker.model.Folder.<init>(Unknown Source:2)
        at com.esafirm.imagepicker.features.fileloader.DefaultImageFileLoader$ImageLoadRunnable.run(DefaultImageFileLoader.java:166)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:919)
```

## Specifications

  - Image Picker Version: 2.4.1
  - Android OS: Android 10
  - Phone: Galaxy s20

**Comment by design365** [comment]
yes, my app is also have this error ,not only android10, but also ,android9

like this: 
java.lang.NullPointerException: Parameter specified as non-null is null: method c.e.a.b.d.a$a.run, parameter name
	at c.e.a.e.b.&lt;init&gt;(Unknown Source:2)
	at c.e.a.b.d.a$a.run(:21)
	at java.util.concurrent.ThreadPoolExecutor.processTask(ThreadPoolExecutor.java:1187)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:784)

**Comment by lillogoal** [comment]
me too

**Comment by HeinerGomez** [comment]
it happens the same to me, in this case, what should be done?.


Fatal Exception: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter folderName
       at com.esafirm.imagepicker.model.Folder.<init>(:2)
       at com.esafirm.imagepicker.features.fileloader.DefaultImageFileLoader$ImageLoadRunnable.run(DefaultImageFileLoader.java:166)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:919)

**Comment by businessworksolution** [comment]
i used updation version. but crash is comming 


java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter folderName
        at com.esafirm.imagepicker.model.Folder.<init>(Unknown Source:2)
        at com.esafirm.imagepicker.features.fileloader.DefaultImageFileLoader$ImageLoadRunnable.run(DefaultImageFileLoader.java:166)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:919)

**Comment by longquangpham90** [comment]
Me to, but can you use version 1.4.0 it very good

**Comment by abxxba** [comment]
i downgraded from 2.4.1 to 2.4.0 and the picker started working , it seems to fix the crashes in android 9 and 10, might not be a long term solution though.

**Comment by esafirm** [comment]
This should be fixed in `2.4.2` by this https://github.com/esafirm/android-image-picker/pull/316
Closing it for now. Will re-open if the issue still persists

**Comment by Farhandroid** [comment]
Thanks a lot ...
